sdk/cognitiveservices/cognitiveservices-luis-authoring/src/lUISAuthoringClientContext.ts

```diff
@@ -43,7 +39,7 @@ export class LUISAuthoringClientContext extends msRest.ServiceClient {
 
     super(credentials, options);
 
-    this.baseUri = "{Endpoint}/luis/authoring/v3.0-preview";
```

Comment:
Since we are now moving out of preview API, the package version should be 4.0.0 unless you want to retain the preview in SDKs for some reason